docs/docs/frequently-asked-questions.mdx

```diff
@@ -295,6 +252,12 @@ guarantees and are not recommended but may fit your use case temporarily:
 - using the internal FAB ModelView API (to be deprecated in Superset)
 - altering the source code in your fork
 
+### How can I see usage statistics (e.g., monthly active users)?
+
+This functionality is not included with Superset, but you can extract and analyze Superset's application
+metadata to see what actions have occurred.  One company has written about [how they analyzed Superset
+usage, including example queries](https://engineering.hometogo.com/monitor-superset-usage-via-superset-c7f9fba79525).
```

Comment:
We should mention superset's action log that by default is sent to the `logs` table in the metadata db? I guess the blog post covers that somewhat

Reply:
Good idea, I just added a sentence re: `logs` table.